drivers/at86rf2xx/at86rf2xx_netdev.c

```diff
@@ -442,6 +442,14 @@ static int _get(gnrc_netdev_t *device, netopt_t opt, void *val, size_t max_len)
             ((uint8_t *)val)[0] = at86rf2xx_get_chan(dev);
             return sizeof(uint16_t);
 
+        case NETOPT_CHANNEL_PAGE:
+            if (max_len < sizeof(uint16_t)) {
```

Comment:
I'm sorry if I'm missing something here, but why do we expect a `uint16_t` pointer here? Especially as we're only using the half of it.